connexion/utils.py

```diff
@@ -131,8 +131,21 @@ def is_json_mimetype(mimetype):
     :type mimetype: str
     :rtype: bool
     """
-    maintype, subtype = mimetype.split('/')  # type: str, str
-    return maintype == 'application' and (subtype == 'json' or subtype.endswith('+json'))
+    simple_mime_type, optional_parameter = mimetype.split(";") if \
+        ";" in mimetype else \
+        (mimetype, None)
+    if "/" not in simple_mime_type:
+        return False
+    maintype, subtype = simple_mime_type.split('/')  # type: str, str
```

Comment:
same here: `partition(..)` will always return 3 values, so we can get rid of the `if`

Reply:
@hjacobs Very elegant. Thanks. I updated.